packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -875,6 +892,72 @@ Accounts.prototype.signTransactionWithSignature = function signTransactionWithSi
     });
 };
 
+/**
+ * combineSignatures combines RLP encoded raw transaction strings.
+ * combineSignatures compares transaction before combining, and if values in field are not same, this throws error.
+ * The result of deocdeTransaction, feePayer can be '0x'(default value) when there is not fee payer's information(address of feePayer and feePayerSignatures)
+ * In this case, feePayer field doesn't have to be compared with other transaction.
+ * 
+ * @method combineSignatures
+ * @param {Array} rawTransactions The array of raw transaction string to combine.
+ * @return {Object}
+ */
+Accounts.prototype.combineSignatures = function combineSignatures(rawTransactions) {
+  let decodedTx
+  let senders = []
+  let feePayers = []
+  let feePayer
+
+  if (!_.isArray(rawTransactions)) throw new Error(`The parameter of the combineSignatures function must be an array of RLP encoded transaction strings.`)
+  
+  for (const raw of rawTransactions) {
+    const { senderSignatures, feePayerSignatures, decodedTransaction } = extractSignatures(raw)
+
+    senders = senders.concat(senderSignatures)
+    feePayers = feePayers.concat(feePayerSignatures)
+    
+    if (decodedTx) {
+      let isSame = true
+      const keys = Object.keys(decodedTx)
+      for (const key of keys) {
+        if (key === 'v' || key === 'r' || key === 's' || key === 'signatures' || key === 'payerV' || key === 'payerR' || key === 'payerS' || key === 'feePayerSignatures') {
+          continue
+        }
+
+        // feePayer field can be '0x' when after sender signs to trasnaction.
+        // For handling this, if feePayer is '0x', don't compare with other transaction
+        if (key === 'feePayer') {
+          if (decodedTx[key] === '0x') {
+            decodedTx[key] = decodedTransaction[key]
+            feePayer = decodedTx[key]
```

Comment:
Is it okay for the case `decodedTx['feePayer'] !== '0x'`?
In this case, `feePayer` is null.
I think we should set `feePayer` at L945.

Reply:
In L945, if there is feePayer information with index 0 of rawTransaction, then there will be existed feePayer information in decodeTx.
But if index 0 of rawTransaction is sender's rawTransaction, then feePayer will be '0x'.
So if decodeTx['feePayer'] is not '0x', address of feePayer will be compared at L938.

Reply:
got it.